taqueria-plugin-ligo/compile.ts

```diff
@@ -57,20 +56,15 @@ const compileContract = (opts: Opts) =>
 				});
 			});
 
-const compileAll = (parsedArgs: Opts): Promise<{ contract: string; artifact: string }[]> => {
-	// TODO: Fetch list of files from SDK
-	return glob(
-		['**/*.ligo', '**/*.religo', '**/*.mligo', '**/*.jsligo'],
-		{ cwd: parsedArgs.config.contractsDir, absolute: false },
-	)
+const compileAll = (parsedArgs: Opts): Promise<{ contract: string; artifact: string }[]> =>
+	Promise.all(getContracts(/ligo|religo|mligo|jsligo$/, parsedArgs.config))
```

Comment:
May I suggests a slight change? Into this: `/\.(ligo|religo|mligo|jsligo)$/`

Just an extra dot

Reply:
Yep, good catch. Thanks!